src/test/java/com/snowflake/kafka/connector/internal/ConnectionServiceIT.java

```diff
@@ -99,7 +99,7 @@ public void createConnectionService() {
         });
 
     SnowflakeURL url = TestUtils.getUrl();
-    Properties prop = InternalUtils.createProperties(TestUtils.getConf(), url.sslEnabled());
+    Properties prop = InternalUtils.createProperties(TestUtils.getConf(), url.sslEnabled(), 0);
```

Comment:
could we add a test to verify non-zero timeout?

Reply:
[sagarrao12](https://github.com/sagarrao12)